scripts/gui/microbe_hud.mjs

```diff
@@ -410,7 +440,27 @@ function updateHoverInfo(vars){
 
             // Debug print version
             // Panel.appendChild(document.createTextNode(key + ": " + vars[key]));
-            panel.appendChild(document.createTextNode(vars[key]));
+            // panel.appendChild(document.createTextNode(vars[key]));
+
+            const values = vars[key].split(":");
```

Comment:
Maybe it would be easier to change the C++ code to send this data as JSON (similar to the patch map) instead of doing string stuff here.